### DISCUSSION TITLE
EQ RAT

### DISCUSSION THREAD

**Discussion by AbbyANoble** [TeamBasedInquiryLearning/precalculus#320] [General]
Author: @AbbyANoble 
Reviewer: @tdegeorge 

Link to RAT: https://www.overleaf.com/read/frcwrfzjjpbk#414b7e 

Link to Readiness: [Readiness Assurance](https://teambasedinquirylearning.github.io/precalculus/2024/instructor/EQ.html#EQ-readiness) 

Note: There were 11 objectives in the readiness. To get to 10 questions, Objective 10 (Simplify square roots of positive numbers) doesn't have its own question, but is instead needed for objectives 6 and 11.

**Comment by tdegeorge** [comment]
For some reason, I can't view this file. I am getting a weird message about how a PDF can't be produced (see attached)
<img width="668" alt="Screenshot 2024-06-13 at 1 35 44 PM" src="https://github.com/TeamBasedInquiryLearning/precalculus/assets/143553140/7c919981-300d-4020-bb82-b7d4b7db7f09">

**Reply by AbbyANoble** [reply]
I found a typo and it's working now!

**Comment by tdegeorge** [comment]
The answer key (at the beginning) does not have the answers in it.
1) B
2) C
3) B
4) D
5) C
6) C
7) B
8) D
9) B
10) C

**Comment by tdegeorge** [comment]
For 6: How about 2 sqrt 44 as another answer? Sometimes students don't pull out the greatest square.

I also see students sometimes forget to take the square root. Like, they will get to 2 sqrt 44 and then see that 44 is 4 times 11 and then try to simplify from there. But because they forget to take the square root of 4, they get 8 sqrt 11.

**Comment by AbbyANoble** [comment]
Fixed and linked in #417
